iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/execution/config/executor/ClusterConfigTaskExecutor.java

```diff
@@ -1669,7 +1671,9 @@ public SettableFuture<ConfigTaskResult> createPipe(CreatePipeStatement createPip
     SettableFuture<ConfigTaskResult> future = SettableFuture.create();
 
     // Validate pipe name
-    if (createPipeStatement.getPipeName().startsWith(PipeStaticMeta.SYSTEM_PIPE_PREFIX)) {
+    if (!(DataRegionConsensusImpl.getInstance() instanceof PipeConsensus
```

Comment:
This is the normal path of pipe creation, thus it cannot be restricted to "PipeConsensus". If you want to let the consensus pipe pass, you can call the "configNodeClient" directly or just prohibit the subscription pipe. The former one is more recommended because it can avoid users from manully creating consensus pipes.

Reply:
I know what you mean, this was mostly written ad hoc for quick testing,  I will fix it